examples/client/client/events.nim

```diff
@@ -5,7 +5,7 @@ import # client libs
   ./common
 
 import # status libs
-  ../../../status/[client, tokens]
+  ../../../status/[api, tokens]
```

Comment:
This is an example where there's a mix of importing the api and non-api modules. See my main comment above; again, I understand this PR is a first step in the reorg/refactor.

Reply:
Agreed. In the future, once we change the root `status.nim` to re-export `/status/api.nim` (which re-exports `/status/api/*.nim`), consuming libraries would only want to `import status` (see https://github.com/status-im/nim-status/pull/254#issuecomment-891583449). However, if a highly-motivated consumer wanted to utilise procs not exposed from importing `status` in itself, nothing is stopping them from importing and utilising individual modules inside of `nim-status`.